src/Widgets/AddressWidget/AddressWidgetComponent.js

```diff
@@ -66,3 +33,84 @@ const Logo = Scrivito.connect(() => {
     </Scrivito.LinkTag>
   );
 });
+
+const Address = Scrivito.connect(({ addressWidget }) => {
+  const localityRegionPostalCode = [
+    addressWidget.get('locationLocality'),
+    addressWidget.get('locationRegion'),
+    addressWidget.get('locationPostalCode'),
+  ].filter(n => n).join(' ');
+
+  const lines = [
+    addressWidget.get('locationName'),
+    addressWidget.get('locationStreetAddress'),
+    localityRegionPostalCode,
+    addressWidget.get('locationCountry'),
+  ].filter(n => n);
+
+
+  if (!lines.length) {
+    return (
+      <InPlaceEditingPlaceholder>
+        Provide the location in the address widget properties.
+      </InPlaceEditingPlaceholder>
+    );
+  }
+
+  const breakedLines = lines.reduce((array, value) => {
+    array.push(value);
+    array.push(<br key={ value } />);
+    return array;
+  }, []);
+
+  return <p>{ breakedLines }</p>;
+});
+
+function Table(values) {
+  const columns = Object.keys(values).map(name => {
+    return <TableColumn key={ name } name={ name } value={ values[name] } />;
+  });
+
+  if (columns.length === 0) {
+    return null;
+  }
+
+  return (
+    <table>
+      <tbody>
+        { columns }
+      </tbody>
+    </table>
+  );
+}
+
+function TableColumn({ name, value }) {
+  const localization = {
```

Comment:
I would extract LOCALIZATION and LINK_PREFIXES so that these objects only get created once, not for every `TableColumn` method call.